ani-cli-win

```diff
@@ -213,7 +229,9 @@ open_episode () {
 	printf "Getting data for episode %d\n" $episode
 
 	dpage_link=$(get_dpage_link "$anime_id" "$episode")
+	# printf "%s\n" "$dpage_link"
 	video_url=$(get_links "$dpage_link")
+	# printf "%s\n" "$video_url"
 
 	case $video_url in
```

Comment:
Can u remove from line 236 to 247.. these lines are useless.. it doesn't do anything..

Reply:
Also from line 92 to 95.. cant't we use jq??..
Because these line will be helpful in new anime.. but in older anime.. it will produce useless link..
I know adding jq as dependency will be a problem in windows.. but if the script is not up to the task then it will make the windows user ditch up the idea of ani-cli

Reply:
Oh, sure!
Also, I could make it so it only uses jq if the user has jq installed, would that be an alright compromise? I'm not sure what the best UX decision is.

Reply:
No. Definitely don't support both. If jq is needed elsewhere in the windows branch already, you might as well port the parsing to jq, but if windows got spared with that dependency then don't change it.
Installing software on windows is more of a hassle and it is safe to assume our average windows user to be less technically literate than our average linux user.
Removing the comments does make sense tho